app/src/main/res/xml/status_widget_info.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:description="@string/status_widget_description"
+    android:initialKeyguardLayout="@layout/status_widget"
+    android:initialLayout="@layout/status_widget"
+    android:minWidth="40dp"
```

Comment:
Maybe move to dimensions? Is 40dp the standard? Please link

Reply:
I will add `min_widget_width` and `min_widget_height` into `res/values/dimens.xml`. OK?

Reply:
I would just call it `widget_min_size` and have one value.